Makefile

```diff
@@ -360,11 +360,7 @@ urltests:
 
 .PHONY: int_test
 int_test:
-	make -j 1 k8stest integrationtest ingresstests evhtests vippernstests dedicatedevhtests dedicatedvippernstests oshiftroutetests bootuptests multicloudtests advl4tests namespacesynctests servicesapitests npltests misc dedicatedvstests hatests calicotests ciliumtests helmtests infratests urltests gatewayapitests
-
-.PHONY: multi_tenancy
-multi_tenancy:
-	make -j 1 multitenancytests
```

Comment:
why are we removing this target?

Reply:
This is redundant to multitenancytests target